Source/UniversalAnalyticsHttpWrapper/EventTracker.cs

```diff
@@ -83,5 +86,31 @@ public async Task<TrackingResult> TrackEventAsync(IUniversalAnalyticsEvent analy
 
             return result;
         }
+
+        /// <summary>
+        /// Adds "raw" (custom) payload to the hit data.
+        /// See https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters for parameters list. For example AddToCustomPayload("aip", "1") to enable IP anonymization.
+        /// If parameter was already added, it's value will be replaced with the supplied one
+        /// </summary>
+        /// <param name="name">Google Analytics Measurement Protocol short parameter name. for ex: aip, ds, qt, etc</param>
+        /// <param name="value">Parameter value</param>
+        public /*bool*/ void AddToCustomPayload(string name, string value)
```

Comment:
I think I prefer the ArgumentException in this case. If someone is developing against this they'll find out pretty quickly if they are setting a field they shouldn't this way :)